packages/logger/README.md

```
@@ -0,0 +1,71 @@
+# @app/logger
+
+**THE ONLY LOGGING SYSTEM ALLOWED IN THIS CODEBASE**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)